### DISCUSSION TITLE
What namespace does InspectContainerData.Namespace describe?

### DISCUSSION THREAD

**Discussion by thediveo** [containers/podman#15589] [Q&A] [answered]
What does [`InspectContainerData.Namespace`](https://github.com/containers/podman/blob/72f4c77139adf61bc39890e08ed9b86c365ae09b/libpod/define/container_inspect.go#L686) describe? Is it even related to Linux kernel namespaces or is it for namespacing container names? I've searched the documentation but didn't found anything that seems to be really closely related to this property of containers when inspecting them. Can someone please shed more light on this?

**Comment by mheon** [comment, marked as answer]
This is, as you surmised, not a kernel namespace, but a Libpod namespace - an old, mostly unused feature we created when we were planning to integrate Libpod into CRI-O and were going to have to separate Kube containers from the rest of Libpod. I'm unaware of anyone actually using them... We should consider deprecating and removing for 5.0.

**Reply by thediveo** [reply]
Thank you very much for the explanation. One thing less to consider in my namespace and ccontainer diagnosis tool.
